types/react/global.d.ts

```diff
@@ -143,6 +143,199 @@ interface SVGTSpanElement extends SVGElement { }
 interface SVGUseElement extends SVGElement { }
 interface SVGViewElement extends SVGElement { }
 
+interface HTMLElementTagNameMap {
+    "a": HTMLAnchorElement;
+    "abbr": HTMLElement;
+    "address": HTMLElement;
+    // TODO: @newyankeecodeshop Any idea why some of these don't exist above?
```

Comment:
cc @newyankeecodeshop ☝️ React does appear to support them (although I’m not immediately clear on where the definitive list of supported tags lives).

Reply:
@newyankeecodeshop The question still stands, but is no longer relevant for this PR.

Reply:
@alloy  I can't remember for sure, but I believe I added all the ones that were in the TypeScript DOM lib at the time, which was probably in the TypeScript 2.2-2.4 timeframe. I suspect more of the HTML 5 interfaces got added over time, and there's nobody specifically keeping this file up to date as the TS Dom lib evolves.

Reply:
Makes sense (alas), thanks 👍